app/Console/Commands/PollTwitter.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\User;
+use Illuminate\Console\Command;
+use Illuminate\Support\Facades\Log;
+
+class PollTwitter extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'suggestive:poll-twitter';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Polls Twitter for any tweets with the configured hashtag.';
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        parent::__construct();
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @return mixed
+     */
+    public function handle()
+    {
+        $users = User::where('role', 'owner')
+                     ->whereNotNull('hashtag')
+                     ->get(['id', 'name', 'hashtag', 'last_tweet_id']);
+
+        if ($users) {
+            Log::info('Found owners to poll Twitter for', ['count' => $users->count()]);
+
+            $twitter = app('TwitterClient');
```

Comment:
No, no need to wait; usually it won't wipe the old comments. I'll look through them.

On this one, your comment was:

> It's not an interface, it's an (I don't know what the proper name is) alias to the underlying return, given the TwitterOAuth client doesn't implement an interface.

> Would you suggest that in this instance we would create an app-specific wrapper for it that implements an interface so that we can bind to it?

> This particular solution has always seemed hacky to me (I use the same method in another app I have). The alternative would be to wrap it in a facade?

I'm not sure what you mean for the Interface; why don't you just typehint `Abraham\TwitterOAuth\TwitterOAuth`? If you want `TwitterClient` as a shortcut like you have it here, you can bind one of them like you are right now, and then have the other one alias to it. Ping me here if you want me to write out the code for what that would look like.